### PR TITLE
Bring back column-aligned svc status

### LINKED ISSUE #5157: `hab sup status` output not column aligned
This is a minor bug in hab/hab-sup 0.56.0.  In previous versions the `hab sup status` output was always nicely formatted into columns.  Now I see:

```
root@automate-deployment:/opt/automate-deployment# hab sup status
package                                      type        state  uptime (s)  pid    group
ssd/deployment-service/0.1.0/20180605120749  standalone  up     270         10392  deployment-service.default
chef/automate-elasticsearch/6.2.2/20180527141927  standalone  up  255  10685  automate-elasticsearch.default
chef/automate-postgresql/9.6.8/20180527135607  standalone  up  253  10761  automate-postgresql.default
chef/authz-service/0.1.0/20180531222343  standalone  up  252  10832  authz-service.default
chef/automate-ui/2.0.0/20180604065308  standalone  up  253  10780  automate-ui.default
chef/authn-service/0.1.0/20180531222136  standalone  up  247  10923  authn-service.default
chef/automate-dex/0.1.0/20180527141333  standalone  up  247  10963  automate-dex.default
chef/teams-service/0.1.0/20180531223417  standalone  up  246  11003  teams-service.default
chef/es-sidecar-service/1.0.0/20180531224355  standalone  down  248  <none>  es-sidecar-service.default
chef/compliance-service/1.11.1/20180601230544  standalone  down  244  <none>  compliance-service.default
chef/license-control-service/1.0.0/20180531224529  standalone  up  243  11086  license-control-service.default
chef/local-user-service/0.1.0/20180531224700  standalone  up  243  11098  local-user-service.default
chef/notifications-service/1.0.0/20180527140145  standalone  up  243  11132  notifications-service.default
chef/session-service/0.1.0/20180531223221  standalone  up  241  11386  session-service.default
chef/config-mgmt-service/0.1.0/20180531223821  standalone  down  241  <none>  config-mgmt-service.default
chef/ingest-service/0.1.0/20180531225222  standalone  down  240  <none>  ingest-service.default
chef/data-lifecycle-service/0.0.1/20180531224001  standalone  up  239  11524  data-lifecycle-service.default
chef/automate-gateway/0.1.0/20180601231211  standalone  up  239  11533  automate-gateway.default
chef/automate-load-balancer/0.1.0/20180527135324  standalone  up  237  11617  automate-load-balancer.default
```

This is much harder to read.

### PR DESCRIPTION
**Pull request by stevendanna**
The TabWriter appears to calculate alignment on every flush(). Thus,
if we defer the flush until the end, all of the columns of output will
be aligned.

Fixes #5157

Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by eeyun** [comment]
Beautiful! Great find!

**Comment by stevendanna** [comment]
Note: I tried to avoid restructuring how this code works; however, that resulted in the somewhat odd use of `fold` just to pass the tabwriter through.  Happy to go another direction if anyone has a recommendation.

File: components/hab/src/main.rs [hunk 1]
```diff
@@ -938,8 +939,15 @@ fn sub_svc_status(m: &ArgMatches) -> Result<()> {
                     }
                     Ok((out, rest))
                 })
-                .and_then(|(mut out, rest)| {
-                    rest.for_each(move |reply| print_svc_status(&mut out, reply, false))
+                .and_then(|(out, rest)| {
+                    rest.fold(out, move |mut out, reply| {
+                        print_svc_status(&mut out, reply, false)?;
+                        Ok::<_, SrvClientError>(out)
```
Comment (stevendanna) [inline components/hab/src/main.rs]: The type-checker gets confused without this annotation.

**Comment by baumanj** [comment]
Nice work! Thanks for your continued support, @stevendanna.

**Pull request merged**